issuance/issuance.go

```diff
@@ -461,6 +462,15 @@ func GetIssuerNameID(ee *x509.Certificate) IssuerNameID {
 	return truncatedHash(ee.RawIssuer)
 }
 
+// GetOCSPIssuerNameID returns the IssuerNameID (a truncated hash over the raw
+// bytes of the Responder Distinguished Name) of the given OCSP Response.
+// As per the OCSP spec, it is technically possible for this field to not be
+// populated: the OCSP Response can instead contain a SHA-1 hash of the Issuer
+// Public Key as the Responder ID. The Go stdlib always uses the DN, though.
```

Comment:
This comment makes me wonder - what does this function do if the field is not populated? Panic? Return some sentinel value?

Reply:
As far as I'm aware, it won't panic: it'll return the sentinel value `61424921164540747`, which is the SHA1 hash of empty input. Would we like to catch that somewhere? We could easily catch that the input is empty/nil here in this function, but then it'd be the only one of the `.ID()` functions which returns an error.

See https://go.dev/play/p/-gOaurDWVa5 for demonstration.

Reply:
I wouldn't consider `61424921164540747` a sentinel value, I'd consider it the hash of a sentinel value. :-)

Let's return 0 in that situation. If it ever _does_ come up it will be more clearly a special value than `61424921164540747` would be.

Reply:
Done!